static/templates/settings/organization-profile-admin.handlebars

```diff
@@ -47,6 +47,7 @@
 
         <h3>{{t "Organization logo" }}</h3>
         <p>{{t "A wide image, replacing the Zulip logo in the upper left corner of the Zulip apps." }}</p>
+        <p>{{t "Logo for white background" }}</p>
```

Comment:
Having 2 paragraphs here looks bad.

Reply:
Agreed, but it's how we seem to style this situation elsewhere in settings?

Reply:
We don't really have anything precisely like H3, p, p on 3 separate lines in the body of a settings page.  I think the problem is just having 3 lines of text here, not the CSS.

Reply:
hmm. Any ideas? 
I think the previous version of this (having each logo thing in a separate `<h3>`) wasn't right, but we can drop the commit and wait till e.g. Max comes online if you don't think it's an improvement. 

This situation (`<h3>` followed by a bit of explanation followed by the first label) seems like a natural pattern the styling should support (e.g. another one is:
![image](https://user-images.githubusercontent.com/890911/53615255-f8160480-3b91-11e9-8a50-1e18ff8c6639.png)